src/xdocs/config_coding.xml

```diff
@@ -2180,6 +2207,16 @@ try {
   &lt;property name=&quot;ignoreOverriddenMethods&quot; value=&quot;false&quot;/&gt;
 &lt;/module&gt;
         </source>
+        <p>Example:</p>
+        <source>
+public class Test{
+    public void func1() throws Throwable{} // violation
+}
+
+class Son extends Test{
```

Comment:
Hi @jokowncode this might be a bit picky, but other examples do not use `Son` or other family members as class name. I can see that you want to point out inheritance here, but I fear this might not sound appropriate in some countries.

Other checks use: `class MyClass extends SomeOtherClass` or `class AnnotationLocationCheck extends AbstractCheck` . Would you mind to change this single example?

Reply:
ok, I will change it.

Reply:
@jokowncode , please do not reply "I will", there is no doubt that you will do this.
Please reply "done" (when you pushed changes and ready for review) or any comment to discuss point further, no other comments.

Reply:
Done